### DISCUSSION TITLE
如何下载油猴

### DISCUSSION THREAD

**Discussion by amns66615** [share121/inter-knot#2795] [其他]
我寻思着这脚本儿也搞不到，已经弄了4个脚本儿了，就是点不开。在线等。
`红温`

**Comment by Yulooooo** [comment]
加我q376799522 教你

**Comment by HKLHaoBin** [comment]
![Clip_2024-08-18_21-10-39](https://github.com/user-attachments/assets/0b960467-a29b-4ce7-bc87-97cd62ab713c)

是这个状态吗

**Comment by amns66615** [comment]
能用微信吗？我没有q



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月18日(周日) 晚上9:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] 如何下载油猴 (Discussion #2795)




 
加我q376799522 教你
